static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -162,51 +139,18 @@ type ContextProps = {
 
 export function QuickContextHoverWrapper(props: ContextProps) {
   const api = useApi();
+  const queryClient = useQueryClient();
   const [ishovering, setisHovering] = useState<boolean>(false);
-  const [error, setError] = useState<boolean>(false);
-  const [loading, setLoading] = useState<boolean>(true);
-  const [data, setData] = useState<Group | null>(null);
-  const delayOpenTimeoutRef = useRef<number | undefined>(undefined);
 
   const handleHoverState = () => {
     setisHovering(prevState => !prevState);
   };
 
-  const fetchData = () => {
-    if (!data) {
-      const params = getRequestParams(
-        props.dataRow,
-        props.contextType,
-        props.organization
-      );
-      api
-        .requestPromise(params.path, params.options)
-        .then(response => {
-          setData(response);
-          if (isIssueContext(props.contextType)) {
-            GroupStore.add([response]);
-          }
-        })
-        .catch(() => {
-          setError(true);
-        })
-        .finally(() => {
-          setLoading(false);
-        });
-    }
-  };
-
-  const handleMouseEnter = () => {
-    handleHoverState();
-    delayOpenTimeoutRef.current = window.setTimeout(() => {
-      fetchData();
-    }, DATA_FETCH_DELAY);
-  };
-
-  const handleMouseLeave = () => {
-    handleHoverState();
-    window.clearTimeout(delayOpenTimeoutRef.current);
-  };
+  useEffect(() => {
+    return () => {
+      queryClient.invalidateQueries({queryKey: ['quick-context-issue']});
```

Comment:
If you made your key for the context components like `queryKey: ['quick-context', 'issue', `${props.dataRow['issue.id']}`]` then I think you could change this to `queryKey: ['quick-context']` and it would just wipe them all